Author: Gregor Riepl <onitake@gmail.com>
Description: Remove RPATH from build flags.
  It's not needed on Linux and may cause maintenance problems.
  See: https://wiki.debian.org/RpathIssue
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,7 +39,6 @@
 if(NOT DEFINED LIB_SUFFIX)
     set(LIB_SUFFIX "")
 endif()
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}")
 
 set(CURA_ENGINE_VERSION "master" CACHE STRING "Version name of Cura")
 
